hare/broker.go

```diff
@@ -192,7 +243,7 @@ func (b *Broker) Unregister(id InstanceId) {
 
 	wg.Add(1)
 	b.tasks <- func() {
-		delete(b.outbox, id)
+		b.outbox[id] = nil // mark unregistered (invalid)
```

Comment:
Remind me (I think we already talked about it), why aren't you deleting the outbox?

Reply:
We need to know that a layer is invalid even after registration. Now thinking about it, maybe we can simply use latestLayer in combination with the fact that the outbox contains nothing about this layer. Let's rethink about this after I mark it on a different map.